Sources/WalletConnectUtils/NewKeyedDatabase.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+public class NewKeyedDatabase<Element> where Element: Codable & Equatable {
```

Comment:
not sure what's the difference but what will be the name of the next one NewNewKeyedDatabase? xd
maybe KeyedDatabaseV2?